### PR TITLE
Add Known Issue for Ingress Canary route edge termination

### PR DESCRIPTION
**Pull request by bmcelvee**
Related to https://bugzilla.redhat.com/show_bug.cgi?id=1932401. 

Preview link (last bullet in list): https://deploy-preview-29807--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-7-release-notes.html#ocp-4-7-known-issues

### CODE REVIEW

**Comment by bmcelvee** [comment]
@sgreene570 here's the RN PR!

**Comment by sgreene570** [comment]
I double checked my patch command and it looks good!
Thanks @bmcelvee this looks great.

**Comment by bmcelvee** [comment]
@openshift/team-documentation PTAL, thanks! :sun_with_face:

File: release_notes/ocp-4-7-release-notes.adoc [hunk 1]
```diff
@@ -2125,6 +2125,18 @@ $ rm -f openshift/99_openshift-cluster-api_worker-machineset-*.yaml
 
 * Currently, on clusters with IPsec enabled, {op-system-base-full} 7.9 nodes cannot communicate with {op-system-first} nodes. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1925925[BZ#1925925])
 
+* If you have clusters that expose the default Ingress Controller through an administrator-provisioned external load balancer that redirects all HTTP traffic to HTTPS, you need to patch the new clear-text Ingress Canary route to use edge termination during the 4.6 to 4.7 upgrade process.
```
Comment (ahardin-rh) [inline release_notes/ocp-4-7-release-notes.adoc:2128]: s/need to/must

File: release_notes/ocp-4-7-release-notes.adoc [hunk 2]
```diff
@@ -2125,6 +2125,18 @@ $ rm -f openshift/99_openshift-cluster-api_worker-machineset-*.yaml
...
++
+.Sample patch command
++
+[source,terminal]
+----
+oc patch route/canary -n openshift-ingress-canary -p '{"spec":{"tls":{"termination":"edge","insecureEdgeTerminationPolicy":"Redirect"}}}'
```
Comment (ahardin-rh) [inline release_notes/ocp-4-7-release-notes.adoc:2134]: $ oc

**Comment by ahardin-rh** [comment]
@bmcelvee Just some minor style suggestions from me :+1:

**Comment by bmcelvee** [comment]
@lihongan would you mind reviewing this release note, please? Thank you!

**Comment by vikram-redhat** [comment]
lgtm - docs ack.

**Comment by lihongan** [comment]
It looks great! 
Thank you @bmcelvee

**Comment by bmcelvee** [comment]
@sferich888 since this is going through our change control process, would you mind reviewing and giving this PR an ack? Please and thank you!

**Comment by bmcelvee** [comment]
@sferich888 since this is going through our change control process, would you mind reviewing and giving this PR an ack? Please and thank you! (Just a bump since this went out right before a long weekend :smile_cat:)

**Comment by sferich888** [comment]
@bmcelvee Ack

**Comment by bmcelvee** [comment]
Thank you!

**Pull request merged**